python/examples/atomistic/1-export-atomistic-model.py

```diff
@@ -215,17 +217,45 @@ def forward(
 # the model, and export it to a file:
 
 wrapper = MetatensorAtomisticModel(model.eval(), metadata, capabilities)
-wrapper.export("exported-model.pt")
+wrapper.save("exported-model.pt")
 
 # the file was created in the current directory
 print(glob.glob("*.pt"))
 
 
 # %%
 #
-# Now that we have an exported model, the next tutorial will show how you can use such a
-# model to run `Molecular Dynamics`_ simulation using the Atomic Simulating Environment
-# (`ASE`_).
+# You can reload the file and use it to predict the ``"energy"`` of a
+# :py:class`System <metatensor.torch.atomistic.System>` of atoms.
+
+atomistic_model = load_atomistic_model("exported-model.pt")
+
+system = System(
+    types=torch.tensor([1]),
+    positions=torch.tensor([[1.0, 1, 1]], dtype=torch.float64, requires_grad=True),
+    cell=torch.zeros([3, 3], dtype=torch.float64),
+)
+
+outputs = {
+    "energy": ModelOutput(quantity="energy", unit="eV", per_atom=False),
+}
+options = ModelEvaluationOptions(length_unit="Angstrom", outputs=outputs)
+
+print(atomistic_model([system], options))
+
+# %%
+#
+# Note that even though the `type` of ``wrapper`` and ``atomistic_model`` is different
+# the API of the objects is the same!
+
+print("type wrapper:", type(wrapper))
+print("type atomistic_model:", type(atomistic_model))
+
+# %%
+#
+# Now that we have an exported model that we can load, the next tutorial will show how
+# you can use such a model to run `Molecular Dynamics`_ simulation using the Atomic
+# Simulating Environment (`ASE`_).
```

Comment:
Users are not supposed to call the models like this, especially in the first tutorial.

Reply:
Yeah probably you are right. Should we move it somewhere else? At least for me it these lines where helpful to understand what is going on.

Reply:
I would put something like this in a tutorial for people writing a new "engine" that would call arbitrary models.

Reply:
No, we don't have one of these tutorial yet. I'm happy to fix the reshaping error, but I think it is more important for this tutorial to be clear and not confusing. If we can find a way to do this while also testing that the model actually runs, that would be ideal!

Reply:
Agree. Adding a test might be a good idea. I will add one.